src/rgw/rgw_common.h

```diff
@@ -143,6 +144,10 @@ using ceph::crypto::MD5;
 #define RGW_ATTR_CRYPT_CONTEXT  RGW_ATTR_CRYPT_PREFIX "context"
 #define RGW_ATTR_CRYPT_DATAKEY  RGW_ATTR_CRYPT_PREFIX "datakey"
 
+/* SSE-S3 Encryption Attributes */
+#define RGW_ATTR_BUCKET_ENCRYPTION RGW_ATTR_PREFIX "encryption."
+#define RGW_ATTR_BUCKET_ENCRYPTION_SSE_S3_KEY_ID RGW_ATTR_BUCKET_ENCRYPTION "sse-s3-key-id"
```

Comment:
the trailing `.` seems odd for the `"encryption."` policy attribute

this looks similar to `#define RGW_ATTR_CRYPT_PREFIX   RGW_ATTR_PREFIX "crypt."` above, but we don't use `RGW_ATTR_CRYPT_PREFIX` directly as an attribute name

how about something like this?
```c++
#define RGW_ATTR_BUCKET_ENCRYPTION_PREFIX RGW_ATTR_PREFIX "sse-s3."
#define RGW_ATTR_BUCKET_ENCRYPTION_POLICY RGW_ATTR_BUCKET_ENCRYPTION_PREFIX "policy"
#define RGW_ATTR_BUCKET_ENCRYPTION_KEY_ID RGW_ATTR_BUCKET_ENCRYPTION_PREFIX "key-id"
```

Reply:
This looks better Casey. I will make the necessary changes along with other suggestions.